rasa/server.py

```diff
@@ -599,6 +606,58 @@ def create_app(
 
         return response.json(response_body)
 
+    @app.post("/conversations/<conversation_id>/inject-intent")
```

Comment:
how about:
```suggestion
    @app.post("/conversations/<conversation_id>/trigger_intent")
```

I think in general we decided for snake_cased endpoints, didn't we?